onnxruntime/core/providers/webgpu/math/matmul_utils.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/inlined_containers.h"
+#include "core/providers/cpu/tensor/utils.h"
+#include "core/providers/webgpu/shader_helper.h"
+
+namespace onnxruntime {
+namespace webgpu {
+
+inline TensorShape BuildTempShapeVector(const TensorShape& shape, const int64_t dim1, const int64_t dim2, const int components) {
```

Comment:
if this will be a general utility function, a more descriptive name or some documentation would be good.